src/_includes/header.html

```diff
@@ -50,6 +50,10 @@
                  alt="External link."></a>
         </li>
         <li><a href="{{ site.baseurl }}/documentation/sdks/python/">Python SDK</a></li>
+<!--         <li><a href="{{ site.baseurl }}/documentation/sdks/pydoc/{{ site.release_latest }}/" target="_blank">Python SDK API Reference <img src="{{ site.baseurl }}/images/external-link-icon.png"
```

Comment:
Because this points to released docs and we don't have a released pydoc yet so it creates the link error in the tests.